src/solvers/prop/prop_conv_solver.h

```diff
@@ -112,7 +112,7 @@ class prop_conv_solvert : public prop_convt, public solver_resource_limitst
   bool post_processing_done = false;
 
   // get a _boolean_ value from counterexample if not valid
-  virtual bool get_bool(const exprt &expr, tvt &value) const;
+  virtual optionalt<bool> get_bool(const exprt &expr) const;
```

Comment:
Maybe this method can actually receive proper documentation? It doesn't at all discuss the error case, and the "if not valid" lacks a subject.

Reply:
fixed